module/builder/collection/config.go

```diff
@@ -28,6 +28,12 @@ type Config struct {
 	// UnlimitedPayer is a set of addresses which are not affected by per-payer
 	// rate limiting.
 	UnlimitedPayers map[flow.Address]struct{}
+
+	// MaxCollectionByteSize is the maximum byte size of a collection.
+	MaxCollectionByteSize uint
```

Comment:
any reason why one is `uint` and other `uint64`?

Reply:
not really, I think anyway is better to avoid `uint` in case we run the protocol on 32bits machines in the future.

Reply:
I changed it to uint64 👍